README.md

```diff
@@ -4,9 +4,11 @@ This is the website for [Savas Labs](http://savaslabs.com).
 
 The site is built using Jekyll. To run locally:
 
-1. Clone the repo
-2. `bundle install`
-3. `bundle exec jekyll serve --config _config.yml,_config.test.yml,_config.dev.yml`
+1. Make sure [Bundler](http://bundler.io) and [Jekyll](http://jekyllrb.com/docs/installation/) are installed
```

Comment:
:rose:, include the commands as a sub bullter: 'gem install bundler, gem install jekyll' or whatever they are.